lib/rpmsg/rpmsg_virtio.c

```diff
@@ -569,14 +569,16 @@ static void rpmsg_virtio_rx_callback(struct virtqueue *vq)
 	uint16_t idx;
 	int status;
 
-	metal_mutex_acquire(&rdev->lock);
-
-	/* Process the received data from remote node */
-	rp_hdr = rpmsg_virtio_get_rx_buffer(rvdev, &len, &idx);
+	while (true) {
```

Comment:
nitpicking, to be aligned with the other infinite loop in the lib 
```suggestion
	while (1) {
```

Reply:
Sure, changed.